lib/inject.js

```diff
@@ -140,6 +138,25 @@ function injectPa11y(window, options, done) {
 		return false;
 	}
 
+	function hiddenAreasContainsElement(element, hiddenSelectors) {
```

Comment:
Ignore I reckon, I'm not sure if it's possible to simplify. At least not easily. I'd rather have the bug fixed and address complexity later :)